starter-code/src/App.js

```diff
@@ -1,18 +1,123 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import FoodBox from './components/FoodBox'
+import Search from './components/Search'
+import TotalsCheckout from './components/TotalsCheckout'
+import foods from './foods.json'
+import { v4 as uuidv4 } from 'uuid';
+
+const newFoods = foods.map(food => {
+
+food['id'] = uuidv4()
+
+return food
+})
 
 class App extends Component {
+
+state = {foods:newFoods,
+         selectedFoods:newFoods,
+         form: false,
+         search:'',
+         name: '',
+         calories: 0,
+         image: '',
+         quantity: 0,         
+        }
+
+
+handleSearchInput = (e) => {
+const selectedFoods = this.state.foods.filter((food) => {
+      return food.name.toLowerCase().includes(e.target.value);
+    });
+this.setState({
+      selectedFoods:selectedFoods,
+      search:e.target.value
+    });
+}
+
+handleInput = (e) => {
+  let {name, value} = e.target
+    this.setState({[name]: value})
+}
+
+addFood = () => {
+if (this.state.form === false) {
+this.setState({form:true})
+}
```

Comment:
I tried it with the following syntax and it works as well `addFood = () => {
  this.setState({form: !this.state.form})
}`